### ISSUE TITLE
Accessibility swarm on the Bulletin component

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2384]
**Is your feature request related to a problem? Please describe.**
Carry out a `two devs` accessibility swarm for the bulletin component once it is completed.

This could be done when integrating the alpha component locally in Simorgh.

**Describe the solution you'd like**
Go through the [accessibility checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
Blocked on https://github.com/bbc/psammead/issues/2382

**Comment by DenisHdz** [comment]
Block on https://github.com/bbc/psammead/issues/2622

**Comment by ghost** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
👀 = Review with UX

## Checklist - Manual ##

  - ### Design  ✅

  - ### Color contrast   ✅

  - ### Color blindness  ✅

  - ### High Contrast mode PC ✅ 

![radio-hc](https://user-images.githubusercontent.com/46446236/69972732-0a0a4680-151a-11ea-8222-a06f6b9ed796.jpg)
![tv-hc](https://user-images.githubusercontent.com/46446236/69972763-1c848000-151a-11ea-9e4c-41d60e7d646e.jpg)



  - ### Color preferences changed in Firefox   ✅

<img width="1009" alt="Screenshot 2019-11-29 at 11 31 12" src="https://user-images.githubusercontent.com/48688870/70021729-c14aa000-15a2-11ea-83ba-ca4cbdb5a48a.png">



  - ### Text resized to 200%  ✅

  - ### Page zoomed to 200%  ✅

  - ### Heading order ✅

  - ### Icons  ✅
   
  - ### Keyboard navigation: tab order, focus css, trace tab path  ✅

  - ### aXe ✅

  - ### No CSS  ✅

  - ### No Javascript  ✅

  - ### bbc-a11y ✅

### Update
After discussion with UX (Nick) we have decided to add a transparent border around the `CTA` for now and fix forward if we come up with a better solution for the `high contrast mode`

**Comment by ghost** [comment]
What is the expected trace tab path for keyboard accessibility?
### 1
![png (1)](https://user-images.githubusercontent.com/48688870/69871272-67588a80-12c3-11ea-8bd9-ae873b686259.png)

### 2
![png](https://user-images.githubusercontent.com/48688870/69871277-6d4e6b80-12c3-11ea-94ba-cae92e1a96a3.png)

**Comment by ghost** [comment]
Confirmed with @greenc05 it should just be tabbing through the headlines. Therefore Option 1 is the correct tab order.

**Comment by DenisHdz** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
👀 = Review with UX

## Checklist - Assistive Technology ##

Acceptance criteria:
```
Radio Bulletin Promo using a screen reader

    Given I use a screen reader
    And I am viewing a list of Promos
    When I am on an on demand Radio Bulletin
    then announce that you can listen this radio bulletin
    and don't announce the CTA

Radio Bulletin Promo LIVE using a screen reader

    Given I use a screen reader
    And I am viewing a list of Promos
    When I am on a LIVE Radio Bulletin
    then announce that you can listen to this LIVE radio bulletin
    and the live label should be announced first
    and the ‘Live’ label should be read in English, with an English accent
    and it should be pronounced as ‘live’ as in ‘live event’ rather than ‘live’ as in ‘I live here’ 
    and don't announce the CTA

Radio bulletins (both) using a keyboard

    Given I use a keyboard
    And I am viewing a list of Promos
    When I an on a (Live on on demand) Radio Bulletin
    then the tab order should as defined in the link below
    https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5dc4535416883f835e7e78e5

TV Bulletin Promo

    Given I use a screen reader
    And I am viewing a list of Promos
    When I am on an on demand TV Bulletin
    then announce that you can watch this TV bulletin
    and don't announce the CTA

TV Bulletin Promo LIVE

    Given I use a screen reader
    And I am viewing a list of Promos
    When I am on an on a LIVE TV Bulletin
    then announce that you can watch this LIVE TV bulletin
    and the live label should be announced first
    and the ‘Live’ label should be read in English, with an English accent
    and it should be pronounced as ‘live’ as in ‘live event’ rather than ‘live’ as in ‘I live here’ 
    and don't announce the CTA

 TV bulletins (both) using a keyboard

    Given I use a keyboard
    And I am viewing a list of Promos
    When I an on a (Live on on demand) TV Bulletin
    then the tab order should as defined in the link below
    https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5dc4535416883f835e7e78e5
```

 - ###  VoiceOver OS (Mac) (Screen Reader) ✅

   > With Safari (Latest Version) on Mac OS (Latest Version)  

 - ### VoiceOver iOS (iPhone/iPad) (Screen Reader) ✅ 

    > Latest Version with Safari (Latest Version) on iOS (Latest Version)

 - ### TalkBack  (Screen Reader) ✅ 
 
   > Latest Version with Chrome (Latest Version) on Android (Latest Version)

- ### ZoomText (Screen Magnifier with Screen Reader capabilities) ✅

  > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### JAWS  (Screen Reader) ✅

   > Version 17 with Internet Explorer 11 on Windows (XP/Vista/7/8/10) 

- ### Read&Write (Reading Solution) ✅ 

   > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### NVDA  (Screen Reader) ✅

   > Latest Version with Firefox (Latest Version) on Windows (XP/Vista/7/8/10)

**Comment by sadickisaac** [comment]
After discussion with UX (Nick) we have decided to add a transparent border around the `CTA` for now and fix forward if we come up with a better solution for the `high contrast mode in firefox`

**Comment by DenisHdz** [comment]
After reviewing it with UX we found this issue with the Radio bulletin summary padding https://github.com/bbc/psammead/issues/2754.

**Comment by DenisHdz** [comment]
This can be closed now that all the issues/bugs have been fixed.
